prime-router/settings/prod/0034-hhsprotect.yml

```diff
@@ -0,0 +1,32 @@
+# Run ./prime receiver set --input ./settings/prod/0030-hhsprotect.yml --env prod
```

Comment:
Note for this and the next few files:

Any file `prime-router/settings/prod/NNNN-xxx.yml` and/or `prime-router/settings/staging/MMMM-yyy.yml ` are "yamlitos" - little snippets of receiver and/or sender configuration changes/additions/subtractions destined for prod or staging respectively.

These changes are not deployed - they ONLY take effect after the file has been submitted to the corresponding prod or staging API that handles settings changes.  Currently, this is done manually.

So, the yamlitos are only here as a convenience to be reviewed; they are not part of the build.